overlay.d/15fcos/usr/libexec/coreos-check-cgroups.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+# This script checks if the system is still using cgroupsv1
+# and prints a message to the serial console.
+
+# Change the output color to yellow
+warn=$(echo -e '\033[0;33m')
+# No color
+nc=$(echo -e '\033[0m')
+
+motd_path=/etc/motd.d/30_cgroupsv1_warning.motd
+
+# `cat` doesn't know how to interpret escape sequences so workaround
+# it by using `echo` inside the heredoc
+# See: https://stackoverflow.com/a/24701349 
+cat << EOF > "${motd_path}"
+${warn}
+cgroups: This system is still using cgroupsv1.
+cgroups: Consider migrating your system + workloads to cgroupsv2.
+cgroups: See the docs about removing kernel arguments https://docs.fedoraproject.org/en-US/fedora-coreos/kernel-args/ for more information.
```

Comment:
Maybe with the URL on a newline? E.g. something like:

```suggestion
For instructions on how to modify kernel arguments for cgroupsv2, see:
https://docs.fedoraproject.org/en-US/fedora-coreos/kernel-args/
```

I think we might end up having more docs for migrating. We can change the URL then.